server/utils/parseMentions.test.ts

```diff
@@ -15,14 +15,16 @@ it("should return an array of mentions", async () => {
   const document = await buildDocument({
     text: `# Header
 
-@[Alan Kay](mention://user/34095ac1-c808-45c0-8c6e-6c554497de64) :wink:
+@[Alan Kay](mention://m/2767ba0e-ac5c-4533-b9cf-4f5fc456600e/user/34095ac1-c808-45c0-8c6e-6c554497de64) :wink:
```

Comment:
What is the `/m/` signifying? Possible to lose it?

Reply:
Just `mention`. Yes we can lose it.

Reply:
We already have `mention://`, so yes feels duplicative 😄 

Sorry to be so picky, we only get one chance to get this right and then it's kind of stuck in the data forever.

Reply:
Yes. Totally fine :+1:

Reply:
Also, not a blocker but it'd be good to have support utility that enables testing the `document.state` scenario here. Currently, `parseMentions` is tested only against `document.text` and the two scenarios differ in the way that one stores `actorId` but other does not.

Reply:
Mmm yes, we can probably make a test in the factory so that every test document gets the correct `state`